packages/thumbprint-react/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Changed
+
+-   [Major] Update dimensions of `EntityAvatar` and `UserAvatar` at all sizes.
```

Comment:
Can we add more info here explaining the breaking changes? It's helpful for future context and for when Zack does the upgrades in his repos.